build-support/bin/ci.py

```diff
@@ -182,10 +173,10 @@ def get_remote_execution_oauth_token_path() -> Iterator[str]:
     yield tf.name
 
 # -------------------------------------------------------------------------
-# Blacklists
+# Blacklists and whitelists
 # -------------------------------------------------------------------------
 
-def get_blacklisted_targets(filename: str) -> Set[str]:
+def get_listed_targets(filename: str) -> Set[str]:
   return {
     line.strip()
     for line in Path(f"build-support/ci_blacklists/{filename}").read_text().splitlines()
```

Comment:
nit: How feasible would it be to change the name of this folder to `build-support/test_filters`? Now that it also supports acceptance lists as well as denial lists.